project/npda/templates/partials/patient_table.html

```diff
@@ -101,6 +103,12 @@
                         {% endif %}
                         {{ patient.diagnosis_date }}
                     </td>
+                    <td class="px-6 py-4 text-center">
+                            {{patient.last_upload_date|date:"d/m/Y H:m s"}}
+                    </td>
+                    <td class="px-6 py-4 text-center">
```

Comment:
Could be nice to have a brief description or tooltip defining the cohort number here

Reply:
I agree nice idea. Worth raising a separate issue. I have handwritten all our tooltips in css but actually would be better to refactor and replace all these with daisyui ones which are nicer. Again, a separate issue for this is a good idea